src/plugins/discover/public/application/main/components/sidebar/discover_field_search.tsx

```diff
@@ -80,13 +99,54 @@ export function DiscoverFieldSearch({ onChange, value, types }: Props) {
 
   const [activeFiltersCount, setActiveFiltersCount] = useState(0);
   const [isPopoverOpen, setPopoverOpen] = useState(false);
+  const [isHelpOpen, setIsHelpOpen] = useState(false);
   const [values, setValues] = useState<State>({
     searchable: 'any',
     aggregatable: 'any',
     type: 'any',
     missing: true,
   });
 
+  const { docLinks } = useDiscoverServices();
+
+  const { curPageIndex, pageSize, totalPages, startIndex, changePageIndex } = usePager({
+    initialPageSize: FIELD_TYPES_PER_PAGE,
+    totalItems: presentFieldTypes.length,
+  });
```

Comment:
Since the pagination component is removed now, there's a lot of logic/variables you don't need either.